src/core/auth/qgsauthmanager.cpp

```diff
@@ -2297,8 +2298,8 @@ bool QgsAuthManager::updateIgnoredSslErrorsCacheFromConfig( const QgsAuthConfigS
 bool QgsAuthManager::updateIgnoredSslErrorsCache( const QString &shahostport, const QList<QSslError> &errors )
 {
   QMutexLocker locker( mMutex.get() );
-  QRegExp rx( "\\S+:\\S+:\\d+" );
-  if ( !rx.exactMatch( shahostport ) )
+  QRegularExpression rx( QRegularExpression::anchoredPattern( "\\S+:\\S+:\\d+" ) );
```

Comment:
anchored pattern is 5.15+ only -- you'll need to ifdef this in

Reply:
This is not true, only anchoredpattern on QStringView were introduced in 5.15, anchoredPattern on string is 5.12 ;) https://doc.qt.io/qt-5/qregularexpression.html#anchoredPattern-1

Reply:
Ah -- that certainly simplifies things!!

Reply:
I have no clue why the 5.15 version is on top with the qstringview 😆